src/io/serialization/v2.0.0/serialize.ts

```diff
@@ -1,35 +1,24 @@
-import { trim, isEqual } from 'lodash'
+/* eslint-disable camelcase */
+import { trim } from 'lodash'
 
 import Ajv from 'ajv'
 import ajvLocalizers from 'ajv-i18n'
 
-import validateShareable, { errors } from '../../../.generated/latest/validateShareable'
+import validateShareable, { errors } from '../../../.generated/2.0.0/validateShareable'
+import { Convert } from '../../../.generated/2.0.0/types'
```

Comment:
Serializer for the old version should use the types, converters and validators from the old version of schema.